pytools/persistent_dict.py

```diff
@@ -68,6 +68,9 @@ class RecommendedHashNotFoundWarning(UserWarning):
     _HAS_ATTRS = True
 
 
+IS_BIGENDIAN = sys.byteorder == "big"
```

Comment:
```suggestion
_IS_BIGENDIAN = sys.byteorder == "big"
```

Reply:
Done in [49820c7](https://github.com/inducer/pytools/pull/285/commits/49820c79a16672e3aeecea1a715d08fa60978fe1)